src/ucp/wireup/wireup_cm.c

```diff
@@ -439,35 +467,19 @@ static unsigned ucp_cm_client_uct_connect_progress(void *arg)
     UCS_STATIC_ASSERT(ucs_static_array_size(ep_init_flags_prio) > 0);
 
     ucp_wireup_eps_pending_extract(ep, &tmp_pending_queue);
-    for (i = 0; i < ucs_static_array_size(ep_init_flags_prio); ++i) {
-        ep_init_flags = ep_init_flags_prio[i];
 
-        /* Cleanup the previously created UCP EP. The one that was created on
-         * the previous call to this client's resolve_cb */
-        ucp_wireup_cm_ep_cleanup(ep);
+    /* Cleanup the previously created UCP EP. The one that was created on
+     * the previous call to this client's resolve_cb */
+    ucp_wireup_cm_ep_cleanup(ep);
 
-        status = ucp_cm_ep_client_initial_config_get(ep, ep_init_flags,
+    for (i = 0; i < ucs_static_array_size(ep_init_flags_prio); ++i) {
+        status = ucp_cm_ep_client_initial_config_get(ep, ep_init_flags_prio[i],
                                         &cm_wireup_ep->cm_resolve_tl_bitmap,
                                         &key);
         if (status != UCS_OK) {
             goto try_fallback;
         }
 
-        ucp_ep_realloc_lanes(ep, key.num_lanes);
-
-        status = ucp_worker_get_ep_config(worker, &key, ep_init_flags,
-                                          &ep->cfg_index);
-        if (status != UCS_OK) {
-            goto err;
-        }
-
-        ep->am_lane = key.am_lane;
-
-        status = ucp_cm_ep_init_lanes(ep, &tl_bitmap);
-        if (status != UCS_OK) {
-            goto err;
-        }
-
```

Comment:
pls restore this code, just need to add ucp_wireup_cm_priv_space_check before it

Reply:
Done. Restored the code now.